app/client/components/cancel/contributions/contributionsCancellationFlowFinancialSaveAttempt.tsx

```diff
@@ -0,0 +1,110 @@
+import { css } from "@emotion/core";
+import { Button, LinkButton } from "@guardian/src-button";
+import { space } from "@guardian/src-foundations";
+import { SvgArrowLeftStraight } from "@guardian/src-icons";
+import { navigate } from "@reach/router";
+import React, { useState } from "react";
+import {
+  isPaidSubscriptionPlan,
+  MembersDataApiItemContext
+} from "../../../../shared/productResponse";
+import { getMainPlan, isProduct } from "../../../../shared/productResponse";
+import { PRODUCT_TYPES } from "../../../../shared/productTypes";
+import { ContributionUpdateAmountForm } from "../../accountoverview/contributionUpdateAmountForm";
+
+const container = css`
+  & > * + * {
+    margin-top: ${space[6]}px;
+`;
+
+const ContributionsCancellationFlowFinancialSaveAttempt: React.FC = () => {
+  const [showAmountUpdateForm, setShowUpdateForm] = useState(false);
+
+  const onUpdateConfirmed = (updatedAmount: number) => {
+    navigate(`mma_financial_circumstances/saved`, { state: { updatedAmount } });
+  };
+
+  const onReduceClicked = () => {
+    setShowUpdateForm(true);
+  };
+
+  const onCancelClicked = () => {
+    navigate(`mma_financial_circumstances/confirmed`);
+  };
+
+  const onReturnClicked = (
+    event: React.MouseEvent<HTMLAnchorElement, MouseEvent>
+  ) => {
+    event.preventDefault();
+    navigate("/");
+  };
```

Comment:
This felt a little hacky - I was trying to use the `LinkButton` component from `source` to get the right styles, but I don't think there's a great way to interate it with client side routing.